index.md

```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** is a declarative domain specific grammar and syntax that allows you to **annotate** and **validate** JSON documents. JSON Schema enables the confident and reliable use of the JSON data format.
 
+## Mission
+
+Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints based validation rules, while enabling and supporting use cases beyond validation.
```

Comment:
I would like to add something to this that mentions annotation by name, because the spec should ensure interoperability around producing and consuming annotations.  The phrase "supporting use cases beyond validation" is vague and completely open-ended.  Annotations are _how_ additional use cases are supported, and that's what the JSON Schema project is concerned with.

Perhaps just this would do:

> Our mission is to enable the definition of interoperable contracts for JSON data in the form of schemas, using constraints-based validation rules, while enabling and supporting **_further use cases through annotation._**

I think that makes it clear that both validation and annotation are part of the interoperable contracts.

Reply:
Good point. Thinking on this though, I consider the use case where a schema is taken "as is" without an instance, and something is created, like a class or a database table (or 2). Are they still annotations?

I wonder if there's a new class of keywords for new vocabularies which are intended to be used without an instance in mind... maybe "declaration". I guess that could be seen as we're saying it's a "declarative" language. We have the duality being used both with and without an instance. Without an instance, it sort of becomes declarative, but still constraints based.

Reply:
> Thinking on this though, I consider the use case where a schema is taken "as is" without an instance, and something is created, like a class or a database table (or 2). Are they still annotations?

Ooooh... excellent question!  Yeah, I've tended to think of that as a different way of making use of the existing keyword classes (not just annotations - assertions like `type` and applicators like `properties` are part of such cases as well).  But that way of declaring things with a schema is worth calling out explicitly.  I think of these as "generative" use cases (generating an instance from a schema- not necessarily generating a JSON instance but usually a code generator or whatever is still working with the JSON data model which is translated further into Java, SQL, etc.

Definitely worth further thought and clear language.